### ISSUE TITLE
Twloader Donor Rom Menu Wont Show Up

### ISSUE THREAD

**Issue by TisNoor** [Robz8/TWLoader#449]
###### Issues about asking for support for piracy will be closed.
-->
### Your 3DS

Tell us which console do you have:

- [x]  Old 3DS
- [ ]  Old 3DS XL
- [ ]  2DS
- [ ]  New 3DS
- [ ]  New 3DS XL
- [ ]  New 2DS XL

### CFW and configuration

Tell us which CFW do you have:

- [ ]  Boot9Strap (SigHax)
- [ ]  A9LH (Only Sysnand)
- [ ]  A9LH + Emunand
- [x]  Luma3DS
- [ ]  AuReiNAND
- [ ]  Other CFW [specify]
- [ ]  Older CFW
- [ ]  Not using CFW / Using TWLoader in 3DSX format

### Console version

Tell us which version do you have:

- [x]  >= 11.4
- [ ]  < 11.4

### TWLoader version

Tell us which versions you are using:

- [x]  Latest release
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release

### Expected behaviour
When I press select the menu where i choose the donor rom should appear

### Actual behaviour
When I press select nothing happens and my select button is not broken

### Steps to reproduce

1.open twloader
2.press select 
3.

### Stack trace and console log

```
```

**Comment by Jolty95** [comment]
@TisNoor are you 100% sure your select buttom works propertly?
And iirc, it's `start` buttom :P

**Comment by TisNoor** [comment]
yeah im 100 % sure my select button works but when i mash it no menu shows up im gonna just try to down load an older version and see what happens

**Comment by TisNoor** [comment]
yes 100%
i dont know what the problem is

**Comment by Jolty95** [comment]
@TisNoor can you tell us a bit more about your setup? With games do you have and where, an screenshot, or whatever more you would like to show please?

**Comment by Nomelas** [comment]
which theme? I found it only works on dsi and 3ds menu, not ak/wood ot r4
